src/fides/config/consent_settings.py

```diff
@@ -9,6 +9,9 @@ class ConsentSettings(FidesSettings):
     tcf_enabled: bool = Field(
         default=False, description="Toggle whether TCF is enabled."
     )
+    ac_enabled: bool = Field(
+        default=False, description="Toggle whether Google AC Mode is enabled."
+    )
```

Comment:
not a must-have - but is there a relatively straightforward way for us to fail fast if `ac_enabled` is set to `True` while `tcf_enabled` is set to `False`? i _think_ we could have a validator to do that?

just feels like it'd be nice to prevent that misconfiguration before anyone gets too far ahead of themselves. basically, enforcing in the application the assumption you mentioned earlier:
> Assumes TCF must be enabled as a prerequisite.

Reply:
ah a validator is a good idea!